docs/content/concepts/assets/external-assets.mdx

```diff
@@ -0,0 +1,192 @@
+| ------------------------------------------------ | --------------------------------------------------------------------------------------- |
+| <PyObject object="external_assets_from_specs" /> | Create a list of external asset definitions from a list of specs                        |
+| <PyObject object="AssetSpec" />                  | An object that represents the metadata of a particular asset                            |
+| <PyObject object="SensorResult" />               | Result object for a sensor evalation where you pass events to attach to external assets |
+| <PyObject object="DagsterInstance" />            | Object presenting the Dagster instance                                                  |
+
+## Defining External Assets
+
+### A single external asset
+
+The following code declares a single external asset and passes it to <PyObject object="Definitions"/>
+
+```python file=/concepts/assets/external_assets/single_declaration.py
+from dagster import AssetSpec, Definitions, external_asset_from_spec
+
+defs = Definitions(assets=[external_asset_from_spec(AssetSpec("my_asset"))])
+```
+
+### External assets with dependencies
+
+External assets can depend on other external assets. You do this by using the `deps` argument of <PyObject object="AssetSpec" />.
+
+```python file=/concepts/assets/external_assets/external_asset_deps.py
+from dagster import AssetSpec, Definitions, external_assets_from_specs
+
+asset_one = AssetSpec("asset_one")
+asset_two = AssetSpec("asset_two", deps=[asset_one])
+
+defs = Definitions(assets=external_assets_from_specs([asset_one, asset_two]))
+```
+
+## Keeping an external asset's metadata up to date.
+
+Since with external assets Dagster does not control scheduling or materialization, it is up to the user to keep the metadata for an external asset up-to-date. There are a number of mechanisms for doing so:
+
+- REST APIs
+- Sensors
+- Log events in ops
+
+### Using REST API
+
+Dagster OSS exposes a REST endpoint for reporting asset materializations.
+
+Here's an example `curl` command in a shell script.
+```
+curl --request POST \
+  --url https://path/to/instance/report_asset_materialization/{asset_key}\
+  --header 'Content-Type: application/json' \
+  --data '{
+	"metadata" : {
+		"source": "From curl command,
+	}
+}'
+```
+
+Here's an example of invoking this API in Python with `requests`:
+
+```python
+import requests
+
+url = f"https://path/to/instance/report_asset_materialization/{asset_key}"
+payload = { "metadata": { "source": "From python script" } }
+headers = { "Content-Type": "application/json" }
+
+response = requests.request("POST", url, json=payload, headers=headers)
+```
+
+There are also end points for reporting asset observations and asset check evaluations.
+
+### Using Sensors
+
+You can generate events to attach to external assets and provide them to sensors directly, by using the `asset_events` param of \<PyObject object="SensorResult/>.
```

Comment:
```suggestion
You can generate events to attach to external assets and provide them to sensors directly, by using the `asset_events` param of <PyObject object="SensorResult />.
```
something funky happening with the rendering of this PyObject, I think removing the leading / will fix